core/src/models/Schedule.ts

```diff
@@ -302,13 +302,13 @@ export class Schedule extends LoggedModel<Schedule> {
   }
 
   @AfterDestroy
-  static async stopRuns(instance: Schedule) {
+  static async destroyRuns(instance: Schedule) {
     const runs = await Run.findAll({
-      where: { creatorId: instance.id, state: "running" },
+      where: { creatorId: instance.id },
     });
 
     for (const i in runs) {
-      await runs[i].update({ state: "stopped" });
+      await runs[i].destroy();
     }
   }
```

Comment:
👍 I'm OK with this change.  I think we did this for history reasons (eg: this run made this import which set this profile property)... but... it's OK.